state/snapshotStatistics.go

```diff
@@ -55,3 +57,20 @@ func (ss *snapshotStatistics) NewDataTrie() {
 func (ss *snapshotStatistics) WaitForSnapshotsToFinish() {
 	ss.wg.Wait()
 }
+
+// PrintStats will print the stats after the snapshot has finished
+func (ss *snapshotStatistics) PrintStats(identifier string, rootHash []byte) {
+	ss.wg.Wait()
+
+	ss.mutex.RLock()
+	defer ss.mutex.RUnlock()
+
+	log.Debug("snapshot statistics",
+		"type", identifier,
+		"duration", time.Since(ss.startTime).Truncate(time.Second),
+		"num of nodes copied", ss.numNodes,
+		"total size copied", core.ConvertBytes(ss.trieSize),
+		"num data tries copied", ss.numDataTries,
```

Comment:
can we also add a function that will better display high numbers? for example, 12.53M will be displayed as 12530000, and it would be better if we display it as 12,530,000

Reply:
I think it will be better if we add that func to the logger.